supabase/migrations/20231207073048_dbdev_supports_new_package_names.sql

```diff
@@ -0,0 +1,342 @@
+insert into app.package_versions(package_id, version_struct, sql, description_md)
+values (
+(select id from app.packages where package_alias= 'supabase@dbdev'),
+(0,0,5),
+$pkg$
+
+create schema dbdev;
+
+-- base_url and api_key have been added as arguments with default values to help test locally
+create or replace function dbdev.install(
+    package_name text,
+    base_url text default 'https://api.database.dev/rest/v1/',
+    api_key text default 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhtdXB0cHBsZnZpaWZyYndtbXR2Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODAxMDczNzIsImV4cCI6MTk5NTY4MzM3Mn0.z2CN0mvO2No8wSi46Gw59DFGCTJrzM0AQKsu_5k134s'
```

Comment:
not for this PR, but we do need to figure out how to deploy clients without baking in the anon key.